static/templates/invite_subscription.handlebars

```diff
@@ -2,11 +2,11 @@
 <a href="#" class="invite_check_all_button">{{t "Check all" }}</a> |
 <a href="#" class="invite_uncheck_all_button">{{t "Uncheck all" }}</a>
 <div id="invite-stream-checkboxes">
-  {{#each streams}}
-  <label class="checkbox">
-    <input type="checkbox" name="stream" value="{{name}}"
-           {{#unless invite_only}}checked="checked"{{/unless}} /> {{name}}
-    {{#if invite_only}}<i class="icon-vector-lock"></i>{{/if}}
-  </label>
-  {{/each}}
+    {{#each streams}}
+    <label class="checkbox">
+        <input type="checkbox" name="stream" value="{{name}}"
+        {{#unless invite_only}}checked="checked"{{/unless}} /> {{name}}
```

Comment:
Yes I guess tool needs to incorporate singleton html tags in someway. I will try modify the tool.
Thanks for reviewing.